api/v1alpha2/plugins/output/loki_types.go

```diff
@@ -0,0 +1,99 @@
+package output
+
+import (
+	"fmt"
+	"kubesphere.io/fluentbit-operator/api/v1alpha2/plugins"
+	"kubesphere.io/fluentbit-operator/pkg/utils"
+)
+
+// +kubebuilder:object:generate:=true
+
+// The loki output plugin, allows to ingest your records into a Loki service.
+type Loki struct {
+	// Loki hostname or IP address.
+	Host string `json:"host,omitempty"`
```

Comment:
Host shouldn't be allowed to be empty.
And items having default values should be set to a default value if they're empty.

![image](https://user-images.githubusercontent.com/18525465/103122520-8e96ac80-46bb-11eb-9987-683088187c85.png)

Reply:
If the item is empty, the fluent bit will use the default value, it is no need to set a default value in the config file.